games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -30,10 +30,10 @@ index a67cd14..8493bb9 100644
  
  SRCS =	main.c		\
 -- 
-1.8.3.4
+2.7.0
 
 
-From 142ac2bc985591b85860f6fea82b5161fc4446e9 Mon Sep 17 00:00:00 2001
+From a90cc639c7268bb4ed9118aa691014ad61963161 Mon Sep 17 00:00:00 2001
```

Comment:
This also allows to have a shorter output when I'm working in a branch and wish to check the diffs agains master by running:
```
git diff master .
```
or
```
git diff master diamonds-3.3.1.2.patchset
```

So I would recommend to manually restore those SHA1 checksums that correspond to parts of the patchset that you did not change.

That said, I believe I'm currently the only contributor who cares about this.
I think most contributors also like to have less noise... but don't do act accordingly and keep the newly randomly generated SHA1 checksums from  haikuporter...
I do my part; and I wish others did theirs, but I'm not going to blame anyone for not doing so, of course!